standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -874,6 +876,9 @@ public enum ConfVars {
             "testing only."),
     HMS_HANDLER_INTERVAL("metastore.hmshandler.retry.interval", "hive.hmshandler.retry.interval",
         2000, TimeUnit.MILLISECONDS, "The time between HMSHandler retry attempts on failure."),
+    HMS_HANDLER_PROXY_CLASS("metastore.hmshandler.proxy", "hive.metastore.hmshandler.proxy",
```

Comment:
do we really need a config for that, since  METASTORE_RETRYING_HANDLER_CLASS it the only implementation at this moment?

Reply:
This config will make Meta Store Server more easy to extend other proxy strategy without modifying the main server code, so I highly recommend this config.

Reply:
👍